app/src/main/res/layout/dialog_album_detail.xml

```diff
@@ -244,7 +244,7 @@
                       android:id="@+id/album_details_hidden"
                       android:textSize="17dp"
                       android:textColor="@color/icongrey"
-                      android:layout_marginLeft="55dp"/>
+                      android:layout_marginLeft="56dp"/>
```

Comment:
Can you please tell me why you made changes in the layout?

Reply:
@anantprsd5 The alignment of the textview here wasn't proper, it required a space indentation.

Reply:
@sauravvishal8797 I would suggest you to solve different bugs in separate issues. Fine for now. Thanks.